.github/workflows/finalize-release.yaml

```diff
@@ -13,48 +13,22 @@ on:
 
 jobs:
   finalize:
-    name: Cherry-pick Changelogs to master
+    name: Update master Changelogs
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
         with:
-          ref: master
           fetch-depth: 0
       - name: Set Git Config
         run: |
           git config --local user.email 38288322+imodeljs-admin@users.noreply.github.com
           git config --local user.name imodeljs-admin
-      - name: Cherry-pick changelogs
+      - name: Run update-changelogs.sh
         run: |
-          git cherry-pick $commitid
-
-          # Clean up files that we don't want to cherry-pick changes back
-          for i in **/package.json; do git checkout --ours $i; done
-          git checkout --ours **/version-policies.json docs/changehistory/NextVersion.md
-
-          # Stage explicit files;
-          #   - Files resolved due to taking master's version
-          #   - All changelogs to remove duplicates from master
-          #     - They would be duplicated if the same PR is backported to the release branch
-          git add **/package.json **/version-policies.json common/changes docs/changehistory/NextVersion.md
-
-          commitMessage=$(git log --format=%B -n 1 $commitid)
-          git commit -m "$commitMessage Changelogs"
-
-          exitCode=$?
-          if [ $exitCode -ne 0 ]
-          then
-            echo Failed to cherry-pick changelogs for $commitMessage. Commit id is $commitid
-            exit $exitCode
-          fi
-
-          rush change --bulk --message "" --bump-type none
-          git add .
-          git commit --amend --no-edit
-
-          exit $?
+          npm i -g zx
+          node ./.github/workflows/automation-scripts/update-changelogs.mjs
```

Comment:
typo? `update-changelogs.mjs` ?

Reply:
ah no. the[ zx package](https://github.com/google/zx) calls for .mjs files, not js. So i actually have a typo in the filename. Thanks